src/pages/partySelectionContainer/partySelection/PartySelection.tsx

```diff
@@ -19,16 +19,23 @@ export default function PartySelection({ parties }: Props) {
   const { t } = useTranslation();
   const bodyTitle = t('partySelection.title');
   const bodyDescription = t('partySelection.subTitle');
-
   const [selectedParty, setSelectedParty] = React.useState<Party | null>();
   const [disableBtn, setBtnDisable] = React.useState(true);
```

Comment:
why don't you just set the initialValues of these 2 useState considering the number of parties in input?